### DISCUSSION TITLE
Setting up Email after Installation

### DISCUSSION THREAD

**Discussion by Sthaaz** [chatwoot/chatwoot#5535] [Self hosted]
I just completed the installation of Chatwoot with ./install install.sh command. 
How can i setup email after completing the setup of chatwoot.

**Comment by vishnu-narayanan** [comment]
Please refer https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables#configure-emails

**Comment by AlexandreSoftway** [comment]
I did all the steps above, but it doesn't work well! I'm trying to look for help, but I already see that there are many users with the same problem and they don't have support to help us. This seems like an excellent solution, but I find myself on my own sometimes! I give up?? Can anyone help us?

![image](https://user-images.githubusercontent.com/83480055/196264859-36b7fa93-b932-40ce-ba34-43acb15ee4fc.png)

**Reply by vishnu-narayanan** [reply]
@AlexandreSoftway Looks like there are some errors with the SMTP credentials you have set up. Could you double-check those values?

Feel free to open a separate GitHub issue with more details about your setup, SMTP configuration, and the error logs.

**Comment by AlexandreSoftway** [comment]
Thanks Vishnu,

 

This is the one of many tests that I try:

 



 

De: Vishnu Narayanan ***@***.***> 
Enviada em: terça-feira, 18 de outubro de 2022 11:54
Para: chatwoot/chatwoot ***@***.***>
Cc: AlexandreSoftway ***@***.***>; Mention ***@***.***>
Assunto: Re: [chatwoot/chatwoot] Setting up Email after Installation (Discussion #5535)

 

@AlexandreSoftway <https://github.com/AlexandreSoftway>  Looks like there are some errors with the SMTP credentials you have set up. Could you double-check those values?

Feel free to open a separate GitHub issue with more details about your setup, SMTP configuration, and the error logs.

**Reply by Rishabhh17** [reply]
Did you find any solution @Sthaaz , I am also facing same issue, I am trying to set up th SMTP but it is not sending mail.

**Reply by AlexandreSoftway** [reply]
i give up! i tryed from many ways and nothing! i'm very dissapointed.

**Reply by Rishabhh17** [reply]
Okay @Sthaaz , Actually I was also trying to implement my SMTP configurations. But I had a bad luck, I tried different smtp configs but no luck. Because of that I was not able to add agents.

**Comment by AlexandreSoftway** [comment]
You can add agents and confirm manually on administration page.

 

De: Rishabh Gupta ***@***.***> 
Enviada em: quarta-feira, 16 de novembro de 2022 12:52
Para: chatwoot/chatwoot ***@***.***>
Cc: AlexandreSoftway ***@***.***>; Mention ***@***.***>
Assunto: Re: [chatwoot/chatwoot] Setting up Email after Installation (Discussion #5535)

 

Okay @Sthaaz <https://github.com/Sthaaz>  , Actually I was also trying to implement my SMTP configurations. But I had a bad luck, I tried different smtp configs but no luck. Because of that I was not able to add agents.
